### PR TITLE
PR fixes from #104

### PR DESCRIPTION
**Pull request by adamkaplan**
PR fixes from PR #104 (added after it was merged already)

### CODE REVIEW

File: TrustKit/Pinning/TSKSPKIHashCache.h [hunk 1]
```diff
@@ -34,11 +34,25 @@ typedef NSMutableDictionary<NSData *, NSData *> SpkiCacheDictionnary;
  */
 - (instancetype _Nullable)initWithIdentifier:(NSString * _Nullable)uniqueIdentifier NS_DESIGNATED_INITIALIZER;
 
+/**
+ Get a pin cache for the provided certificate and public key algorithm. The pins
+ are cached so
```
Comment (dbgrandi) [inline TrustKit/Pinning/TSKSPKIHashCache.h]: Seems like you trailed off mid-sentence here.

File: TrustKit/Reporting/TSKReportsRateLimiter.h [hunk 2]
```diff
@@ -20,6 +20,13 @@
  */
 @interface TSKReportsRateLimiter : NSObject
 
+/**
+ Determine if the report should be reported or ignored due to the rate limiting policy.
+
+ @param report The report to check whether or not to rate limit
+ @return True if the report should be ignored under the rate-limiting policy that
```
Comment (dbgrandi) [inline TrustKit/Reporting/TSKReportsRateLimiter.h]: `True` or `YES`

File: TrustKit/TSKPinningValidator.h [hunk 3]
```diff
@@ -41,13 +41,41 @@ typedef NSData* _Nullable(^HashCertificateBlock)(_Nonnull SecCertificateRef cert
...
 ///------------------------------------
 
+
+/**
+ Domain pinning configuration, typically obtained by parseTrustKitConfiguration()
+ */
 @property (nonatomic, readonly, nullable) NSDictionary *pinnedDomains;
+
+/**
+ Set to true to ignore the trust anchors in the user trust store. Only applicable
```
Comment (dbgrandi) [inline TrustKit/TSKPinningValidator.h]: Another `true` vs `YES` thing. I don't have an opinion, just pointing them out here.

File: TrustKit/TSKPinningValidator.h [hunk 4]
```diff
@@ -41,13 +41,41 @@ typedef NSData* _Nullable(^HashCertificateBlock)(_Nonnull SecCertificateRef cert
...
+/**
+ The callback invoked with validation results
+ */
 @property (nonatomic, readonly, nonnull) void(^validationResultHandler)(TSKPinningValidatorResult * _Nonnull result);
 
+/**
+ Initialize an instance of TSKPinningValidatorResult
+
+ @param pinnedDomains Domain pinning configuration, typically obtained by parseTrustKitConfiguration()
+ @param name A name for this instance. It should be unique to differentiate insstances
```
Comment (dbgrandi) [inline TrustKit/TSKPinningValidator.h]: `insstances` (sp)

File: TrustKit/TSKPinningValidator.h [hunk 5]
```diff
@@ -41,13 +41,41 @@ typedef NSData* _Nullable(^HashCertificateBlock)(_Nonnull SecCertificateRef cert
...
+    in logs and cache files.
+ @param ignorePinsForUserTrustAnchors Set to true to ignore the trust anchors in the user trust store
+ @param validationResultQueue The queue use when invoking the validationResultHandler
```
Comment (dbgrandi) [inline TrustKit/TSKPinningValidator.h]: `used`

**Comment by adamkaplan** [comment]
All addressed, thanks!

**Pull request merged**